templates/email/default/templated_email_alert-update.html

```diff
@@ -0,0 +1,31 @@
+[%
+
+title = report.title | html;
+email_summary = "Council update on “" _ title _ "”";
+email_columns = 2;
+
+PROCESS '_email_settings.html';
+
+INCLUDE '_email_top.html';
+
+%]
+
+<th style="[% td_style %][% primary_column_style %]" id="primary_column">
+  [% start_padded_box | safe %]
+  <h1 style="[% h1_style %]">New update on <a href="[% problem_url %]">[% title %]</a></h1>
```

Comment:
It's my understanding that this email could actually contain multiple updates, which is why the original version of this email says "New updates on". I think it's ok to say new updates when there's only one update, so probably best to follow the original template here.